### ISSUE TITLE
Antidote for iPhone X / Xs / Xs Max

### ISSUE THREAD

**Issue by jean-io** [Antidote-for-Tox/Antidote#318]
Hello,

This app has not been updated for iPhones X / Xs / Xs Max:

![2018-11-28 14-25-43-1](https://user-images.githubusercontent.com/4070933/49154957-0b5acf00-f31a-11e8-8079-c05af9b68fec.png)

Thank for taking a look :+1:

**Comment by dvor** [comment]
Yeah, it needs to be updated. Currently I don't have much time to work on Antidote.

**Comment by jean-io** [comment]
ok, no problem.

**Comment by ruuti** [comment]
I could look into this and create a pull request. If I do so, would do have time to check out the fix and maybe create an update or are totally out of time?

**Comment by dvor** [comment]
@ruuti yes, I can take a look at your PR, feel free to submit it.

**Comment by rajatsha90271** [comment]
Is the issue still open? . I can see that the application is already supporting iPhone X / Xs / Xs Max.

**Comment by dvor** [comment]
App is not optimized for iPhone X, which means it has black stripes on top and bottom of screen.
